wsi/drm/drmdisplay.cpp

```diff
@@ -778,6 +781,14 @@ void DrmDisplay::SetPipeCanvasColor(uint16_t bpc, uint16_t red, uint16_t green,
                            canvas_color_prop_, canvas_color);
 }
 
+void DrmDisplay::SetPipeMaxBpc(uint16_t max_bpc) const {
+  if (max_bpc_prop_ == 0)
+    return;
+
+  drmModeConnectorSetProperty(gpu_fd_, connector_, max_bpc_prop_,
```

Comment:
drmModeConnectorSetProperty() does not check to ensure gpu_fd_ is valid which can be a problem if gpu_fd_ were to be invalid.

For now, check gpu_fd_ here or do it the correct way by checking gpu_fd_ in drmModeConnectorSetProperty().

Reply:
I think the fd check is taken care in libdrm.

Reply:
Hopefully so.  Can you please confirm?  This is important for stability.

Reply:
The gpu_fd_ is set during initialization of drmdisplay, which typically would never be wrong.
My suggestion would be to return bool instead of void and check the return value from drmModeConnectorSetProperty instead of assuming that it would always succeed.

Reply:
@harishkrupo That sounds proper.